_locales/de/messages.json

```diff
@@ -139,22 +155,18 @@
         "message": "Zum Speichern anklicken",
         "description": "Hover text for attachment filenames"
     },
-    "fileABug": {
-        "message": "Fehler melden",
-        "description": "Item under the Help menu, takes you to GitHub new issue form"
-    },
     "messagesBelow": {
         "message": "Neue Nachrichten unten",
         "description": "Alt text for button to take user down to bottom of conversation with more than one message out of screen"
     },
-    "installGeneratingKeys": {
-        "message": "Schlüssel werden erzeugt …",
-        "description": ""
-    },
     "resetSession": {
-        "message": "Verschlüsselung zurücksetzen",
+        "message": "Verschlüsselung neu starten",
```

Comment:
Previous one was better. New one says: Restart encryption. Both _restart_ and _encryption_ are poor translations. Will try and revisit German translations myself at some point.

Reply:
You can say in german "Session zurücksetzen"

Reply:
@Dafnik I like your suggestion! If you’d like to contribute directly, please review our strings here: https://www.transifex.com/liliakai/signal-desktop/

I grew up in Switzerland 🇨🇭and it’s been a while since I moved, so my I don’t always trust my Hochdeutsch ⬆🇩🇪:wink: